RGLServerPlugin/CMakeLists.txt

```diff
@@ -40,3 +40,23 @@ target_link_libraries(RGLServerPluginInstance
     ignition-plugin1::ignition-plugin1
     ${RobotecGPULidar}
 )
+
+## Install libraries
+
+find_program(PATCHELF "patchelf")
+if(NOT PATCHELF)
+    message(FATAL_ERROR "'patchelf' executable not found. patchelf is required for patching 'runpath' of RGLServerPlugin libraries.\n"
+            "You can install it via 'sudo apt install patchelf'")
+endif()
+
+install(TARGETS RGLServerPluginInstance RGLServerPluginManager
+    DESTINATION RGLServerPlugin
+)
+install(FILES ${RobotecGPULidar}
+    DESTINATION RGLServerPlugin
+)
+
+# rpath for each RGLServerPlugin lib must be updated to link RobotecGPULidar properly
+install(CODE
+    "execute_process(COMMAND find ${CMAKE_INSTALL_PREFIX}/RGLServerPlugin -name *.so -exec patchelf --set-rpath $ORIGIN:. {} \;)"
+)
```

Comment:
CMake is insane.

Reply:
Isn't there any CMake-specific mechanism for this? Hard to believe this is always done this way.

Reply:
You are right! Fixed